config/locales/de.events.yml

```diff
@@ -20,7 +20,7 @@ de:
       maximum_number_of_participants_exeeded: 'Maximale Teilnehmeranzahl wurde überschritten'
       sending_acceptances: 'Zusagen verschicken'
       sending_rejections: 'Absagen verschicken'
-      address_list: 'E-Mail Addressen'
+      address_list: 'E-Mail-Addressen'
```

Comment:
Exemplary. But 'Addressen' is written with one 'd' :stuck_out_tongue_winking_eye: